packages/documentation/src/pages/about-mdx.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: About MDX
+---
+
+## About MDX
+
+### What is MDX?
+We will be using the .mdx format to create our documents.
+MDX is a format that lets you seamlessly use JSX in your Markdown documents.
+You can import components, like interactive charts or notifs, and export metadata.
+This makes writing long-form content with components a blast 🚀
+
+[https://mdxjs.com/](https://mdxjs.com/)
+
+### Example
+Here is a great page of examples of using MDX
+
+- http://dump.jamesknelson.com/mdxc-playground.html
+
+### Findings on things you can do with MDX and gotchas
+This is just documentations on finding of using MDX in our projects. Please feel
+free to add to this document if you find anything else.
+
+#### Inline components
+You can create components incline inside your MDX files. Below is an example
```

Comment:
`You can use components inline inside` ... `files; Below`

`use` might be better as `demonstrate` or whatever activity we're intending them to do with this helper. Something like "you can demonstrate usage of your component" or something like that. I'm not sure exactly what this is for.

Reply:
This first part might be good as a bulleted list 
```
- You can create ...
- You will need to export ...
- You can write both ...
```